src/lib/SapphireClient.ts

```diff
@@ -23,6 +23,97 @@ export interface SapphirePrefixHook {
 	(message: Message): Awaited<string | readonly string[] | null>;
 }
 
+export interface SapphireClientOptions {
+	/**
+	 * The base user directory, if set to `null`, Sapphire will not call [[SapphireClient.registerUserDirectories]],
+	 * meaning that you will need to manually set each folder for each store. Please read the aforementioned method's
+	 * documentation for more information.
+	 * @since 1.0.0
+	 * @default undefined
+	 */
+	baseUserDirectory?: string | null;
+
+	/**
+	 * The default prefix, if it returns `null`, only mention prefix will trigger the bot's commands.
+	 * @since 1.0.0
+	 * @default null
+	 */
+	defaultPrefix?: string | null;
+
+	/**
+	 * The prefix hook, by default it is a callback function that returns [[SapphireClientOptions.defaultPrefix]].
+	 * @since 1.0.0
+	 * @default () => client.options.defaultPrefix
+	 */
+	fetchPrefix?: SapphirePrefixHook;
+
+	/**
+	 * The internationalization options, defaults to an instance of [[Internationalization]] when
+	 * [[ClientInternationalizationOptions.instance]] is not specified.
+	 * @since 1.0.0
+	 * @default new Internationalization('en-US')
+	 */
+	i18n?: ClientInternationalizationOptions;
+
+	/**
+	 * The client's ID, this is automatically set by the CoreReady event.
+	 * @since 1.0.0
+	 * @default this.client.user?.id ?? null
+	 */
+	id?: string;
+
+	/**
+	 * The logger options, defaults to an instance of [[Logger]] when [[ClientLoggerOptions.instance]] is not specified.
+	 * @since 1.0.0
+	 * @default new Logger(LogLevel.Info)
+	 */
+	logger?: ClientLoggerOptions;
```

Comment:
If this can be an instance as the docs suggest, these types are inaccurate

Reply:
That's `logger.instance`.

Reply:
Then your doc string is misleading